docs/eog/eog-in-practice/cw21/feasibility-stage/fs-outputs-and-outcomes.rst

```diff
@@ -0,0 +1,35 @@
+.. _cw21-fs-outputs-and-outcomes:
+
+CW21 Outputs and Outcomes 
+==========================
+
+Outputs are the things produced during or for the workshop, be it physical or virtual, and outcomes are the changes made possible due to running the event. 
+These should be linked to the event goals.
+Some of these will be the same as the objectives of the workshop. 
+They should include the type of resources that will be available after the event. 
+That is, whether a report will be written, videos of the sessions or any other resources will be made available.
+
+
+Outputs
+--------------------
+
+- Recordings of the sessions shared via the SSI Youtube channel
+- Closed captioning, transcriptions
+- Blog posts from the discussion session
+- All slides and other outputs shared via the conference portal (e.g. Figshare)
+- A highlights blog post on the SSI website
+- A reflections blog post on the SSI website on how we ran it
+- Collaborative notes documents and other unconference infrastructure resources shared
+- Feedback into Event Organisation Guide (e.g. as a case study)
+- Any sponsor outputs (e.g. Tweets, videos, etc.)
+
+
+Outcomes
+--------------------
+
+- Facilitation of new collaborations between participants (e.g. increased participation in FAIR 4RS working group)
+- Outputs are more widely shared, increasing impact of discussions and visibility of CW/SSI
+- Increased accessibility and inclusion
+- Community members who are unable to attend the event are still able to engage with the discussion
+- Impact on virtual events run by others
```

Comment:
Can we detail this 'Impact' <-- do we mean informing how they run their virtual events or more than that?